build.sbt

```diff
@@ -1,3 +1,6 @@
+import sbt.*
```

Comment:
Not required?

Reply:
Addressed in https://github.com/ReactiveMongo/ReactiveMongo/pull/1245/commits/6e239cbfd14ae545ced37d3297fdb5c5cd38a1d7